<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="2" ignored="0" total="3" passed="1">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="70277" started-at="2022-12-23T18:17:22Z" finished-at="2022-12-23T18:18:32Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="70277" started-at="2022-12-23T18:17:22Z" finished-at="2022-12-23T18:18:32Z">
      <class name="tests.LoginTest">
        <test-method status="PASS" signature="setUp()[pri:0, instance:tests.LoginTest@363ee3a2]" name="setUp" is-config="true" duration-ms="1102" started-at="2022-12-23T18:17:21Z" finished-at="2022-12-23T18:17:22Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="PASS" signature="setDriver()[pri:0, instance:tests.LoginTest@363ee3a2]" name="setDriver" is-config="true" duration-ms="5763" started-at="2022-12-23T18:17:22Z" finished-at="2022-12-23T18:17:28Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setDriver -->
        <test-method status="FAIL" signature="loginErrorMsg_TC01()[pri:0, instance:tests.LoginTest@363ee3a2]" name="loginErrorMsg_TC01" duration-ms="7468" started-at="2022-12-23T18:17:28Z" finished-at="2022-12-23T18:17:35Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Please enter your password. ] but found [Please enter your password.]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Please enter your password. ] but found [Please enter your password.]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:453)
at org.testng.Assert.assertEquals(Assert.java:463)
at tests.LoginTest.loginErrorMsg_TC01(LoginTest.java:31)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginErrorMsg_TC01 -->
        <test-method status="PASS" signature="removeDriver()[pri:0, instance:tests.LoginTest@363ee3a2]" name="removeDriver" is-config="true" duration-ms="1792" started-at="2022-12-23T18:17:35Z" finished-at="2022-12-23T18:17:37Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- removeDriver -->
        <test-method status="PASS" signature="setDriver()[pri:0, instance:tests.LoginTest@363ee3a2]" name="setDriver" is-config="true" duration-ms="2657" started-at="2022-12-23T18:17:37Z" finished-at="2022-12-23T18:17:39Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setDriver -->
        <test-method status="FAIL" signature="loginToSF_TC02()[pri:0, instance:tests.LoginTest@363ee3a2]" name="loginToSF_TC02" duration-ms="19866" started-at="2022-12-23T18:17:39Z" finished-at="2022-12-23T18:17:59Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Free trial should be seen expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Free trial should be seen expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertTrue(Assert.java:44)
at tests.LoginTest.loginToSF_TC02(LoginTest.java:43)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginToSF_TC02 -->
        <test-method status="PASS" signature="removeDriver()[pri:0, instance:tests.LoginTest@363ee3a2]" name="removeDriver" is-config="true" duration-ms="1409" started-at="2022-12-23T18:17:59Z" finished-at="2022-12-23T18:18:01Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- removeDriver -->
        <test-method status="PASS" signature="setDriver()[pri:0, instance:tests.LoginTest@363ee3a2]" name="setDriver" is-config="true" duration-ms="2488" started-at="2022-12-23T18:18:01Z" finished-at="2022-12-23T18:18:03Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setDriver -->
        <test-method status="PASS" signature="rememberMe_TC03()[pri:0, instance:tests.LoginTest@363ee3a2]" name="rememberMe_TC03" duration-ms="26865" started-at="2022-12-23T18:18:03Z" finished-at="2022-12-23T18:18:30Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- rememberMe_TC03 -->
        <test-method status="PASS" signature="removeDriver()[pri:0, instance:tests.LoginTest@363ee3a2]" name="removeDriver" is-config="true" duration-ms="1875" started-at="2022-12-23T18:18:30Z" finished-at="2022-12-23T18:18:32Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- removeDriver -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:tests.LoginTest@363ee3a2]" name="tearDown" is-config="true" duration-ms="1207" started-at="2022-12-23T18:18:32Z" finished-at="2022-12-23T18:18:33Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- tests.LoginTest -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
